src/scss/checkly-webapp-src/style/fonts/fonts.scss

```diff
@@ -1,10 +1,4 @@
-@font-face {
-	font-family: 'Inter';
-	font-style: normal;
-	font-weight: 100 900;
-	font-display: optional;
-	src: url('../fonts/Inter/Inter.var.woff2') format('woff2');
-}
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
```

Comment:
@ndom91 Using `import` is a pretty bad perf pattern. I'm not sure where this pops out, but are we sure we want to do this? It potentially delays first paint.

Reply:
Yeah so this is what I mentioned in Slack, not-selfhosting your fonts forces us to do another import, make another https connection (incl tls handshake), etc. 

It kinda goes against "best-practices", but also would save us on bandwidth costs. (This inter file is our biggest bandwidth perpetrator). 

So its up to you guys, I dont really have a strong opinion. Was just trying to improve bandwidth costs with these marketing website changes  :+1: (See discussion with Daniel linked above for more details).

CC: @hlenke